src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--first-day-tue calendar--month-length-29">
+      <div class="day"></div>
```

Comment:
Need to fix everywhere
```suggestion
      <div class="calendar__day"></div>
```